Tests/powershell-helpers.ps1

```diff
@@ -37,18 +37,29 @@ function Test-AppExists($appName) {
   return $null -ne $command
 }
 
-function Test-PowershellModuleInstalled($moduleName, $version) {
-  $command = Get-Module $moduleName -listavailable
+function Import-PowerShellModule ($Name, $MinimumVersion) {
+  Write-Output "Checking version for $moduleName..."
+  $command = Get-Module $Name -listavailable
   if ($null -eq $command) {
-    write-host "Please install $($moduleName): Install-Module -Name $moduleName -Force" -foregroundcolor red
+    write-host "Please install $($Name): Install-Module -Name $Name -Force" -foregroundcolor red
     exit 1
   }
-  if ($null -ne $version) {
-    if ($command.Version -lt [System.Version]::Parse($version)) {
-      write-host "Please install $($moduleName) $version or higher (you have version $($command.Version)): Update-Module -Name $moduleName -Force" -foregroundcolor red
+
+  if ($command.Count -gt 1)
```

Comment:
[nit] Would prefer the open brace on the same line to be consistent with the rest of the file

Reply:
[similar nit] that's always driven me nuts, prefer to align them :P  However, for consistency, I will conform.